components/mediation-admin/org.wso2.carbon.message.processor/src/main/java/org/wso2/carbon/message/processor/service/MessageProcessorAdminService.java

```diff
@@ -755,4 +762,110 @@ public boolean validateAxis2ClientRepo(String input) throws AxisFault {
         }
     }
 
+
+    /**
+     * Gets the message from the associated queue
+     *
+     * @param processorName Message Processor name
+     * @return <code>msg</code> Returns message received from the queue as a string
+     * @throws Exception
+     */
+
+    public String getMessage(String processorName) {
+        SynapseConfiguration configuration = getSynapseConfiguration();
+        MessageConsumer messageConsumer = getMessageConsumer(configuration,processorName);
+        String msg;
+
+        msg = getMessageAsString(messageConsumer);
+        messageConsumer.cleanup();
+        return msg;
+    }
+
+    private String getMessageAsString(MessageConsumer consumer) throws SynapseException {
+        MessageContext messageContext;
+        MessageConsumer messageConsumer = consumer;
```

Comment:
`messageConsumer` is redundant. Can't we use the method param `consumer` in the places where `messageConsumer` is used?